package.json

```diff
@@ -63,7 +63,7 @@
     "@types/object-hash": "^3.0.6",
     "@types/yargs": "^17.0.33",
     "espower-typescript": "^10.0.1",
-    "gapic-node-processing": "^0.1.2",
+    "gapic-node-processing": "^0.1.21-experimental",
```

Comment:
Will we be able to get automated dependency updates to this? If not, is there a tracking bug for updating this once it's not experimental?